templates/vhost/_scriptalias.epp

```diff
@@ -0,0 +1,15 @@
+<%- $aliases = $scriptaliases -%>
+  ## Script alias directives
+<%# Combine scriptalias and scriptaliases into a single data structure -%>
+<%# for backward compatibility and ease of implementation -%>
+<%-  if $scriptalias { $aliases1 = $aliases << { 'alias' => '/cgi-bin', 'path' => $scriptalias } } -%>
+<%-  else { $aliases1 = $scriptaliases } -%>
+<%- $aliases1.each |$salias| { -%>
```

Comment:
I'd consider this for readability. Note that line 1 is then redundant.
```suggestion
<%- $aliases1 = if $scriptalias { [{ 'alias' => '/cgi-bin', 'path' => $scriptalias }] } else { [] } -%>
<%- ($scriptaliases + $aliases1).each |$salias| { -%>
```

Reply:
i think they are updating existing array of hash on condition this new hash will get added to that array ..else it will have old values

Reply:
Done